src/UI/Checkbox.elm

```diff
@@ -66,7 +64,6 @@ type alias Properties msg =
 
 type alias Options =
```

Comment:
I was wondering if we should get rid of this record since it has only one field now and change the 'Checkbox msg` type like this:

```elm
type Checkbox msg = Checkbox (Properties msg) Bool
```

cc: @PedroHLC

Reply:
I would keep it just to maintain API pattern, but the rest of the team may not agree with me...